apps/remix-ide/src/app/tabs/locales/en/udapp.json

```diff
@@ -65,16 +65,18 @@
   "udapp.tooltipText3": "Click to open a bridge for converting L1 mainnet ETH to the selected network currency.",
 
   "udapp._comment_instanceContainerUI.tsx": "libs/remix-ui/run-tab/src/lib/components/instanceContainerUI.tsx",
-  "udapp.deployedContracts": "Deployed Contracts",
+  "udapp.deployedContracts": "Deployed/Unpinned Contracts",
   "udapp.deployAndRunClearInstances": "Clear instances list and reset recorder",
-  "udapp.deployAndRunNoInstanceText": "Currently you have no deployed contracts to interact with.",
-  "udapp.tooltipText6": "Autogenerated generic user interfaces for interaction with deployed contracts",
+  "udapp.deployAndRunNoInstanceText": "Currently you have no unpinned contracts to interact with.",
+  "udapp.tooltipText6": "Autogenerated generic user interfaces for interaction with deployed/unpinned contracts",
 
-  "udapp.savedContracts": "Saved Contracts",
-  "udapp.NoSavedInstanceText": "No saved contracts found.",
-  "udapp.tooltipTextUnsave": "Unsave & move to Deployed Contracts list",
-  "udapp.savedOn": "Saved On",
-  "udapp.filePath": "File Path",
+  "udapp.savedContracts": "Pinned Contracts",
+  "udapp.tooltipTextPinnedContracts": "List of pinned contracts for selected network",
+  "udapp.NoSavedInstanceText": "No contracts pinned yet. Pinned contracts stay after reloading Remix IDE.",
```

Comment:
Also, i think this info will be better in a bootstrap alert. Something like the attached screenshot.
<img width="289" alt="Screenshot 2024-03-08 at 11 45 21" src="https://github.com/ethereum/remix-project/assets/23282059/2e7f30fd-adf4-4c2f-9e89-fb10ab70478e">

Reply:
That can be discussed and come in a different PR

Reply:
Liana wanted it as info instead of warning

Reply:
Yes, we could have it as a bootstrap alert info